src/combineReducers.js

```diff
@@ -74,6 +74,46 @@ function assertReducerSanity(reducers) {
   })
 }
 
+function getFinalReducers(reducers) {
```

Comment:
I’d rather have the argument be `config`, the variable be `reducers`, and the function be `getReducers`.